cmd/manager/main.go

```diff
@@ -44,7 +44,7 @@ func configureLogger() (*zap.Logger, error) {
 func hasRequiredVariables(logger *zap.Logger, envVariables ...string) bool {
 	allPresent := true
 	for _, envVariable := range envVariables {
-		if _, envSpecified := os.LookupEnv(envVariable); !envSpecified {
+		if _, envSpecified := os.LookupEnv(envVariable); !envSpecified { // nolint:forbidigo
```

Comment:
q: I thought we are fine with all files named `main.go` to keep the env var usage? If yes, could we exclude the whole file instead of single lines? Or is that on purpose here?

Reply:
@nammn I considered doing this, but as far as I understand it is only possible to exclude whole linter. It is not an issue right now because we only use `forbidigo` only for env vars right now. But in the future we decide to use `forbidigo` for something else `main.go` will become a blind spot for this linter.

What do you think - still worth adding an exception for main.go?

Reply:
I think its better to skip the whole file for now and in case we add more linters we can revisit this decision.